.travis.yml

```diff
@@ -3,10 +3,11 @@ bundler_args: --without debug release
 before_install:
   - gem install bundler
 rvm:
-  - 1.9.3
-  - 2.0.0
+  - 2.1.9
```

Comment:
Maybe go extra step and give ["2.1.10"](https://www.ruby-lang.org/en/news/2016/04/01/ruby-2-1-10-released/) a chance?